shared/language/language.go

```diff
@@ -8,7 +8,9 @@
 //	3. Consolidates triggers by categories (e.g. automotive brands) if
 //	commonly used across plugins.
 //	4. Summarizes text using the custom rule-based algorithm found in
-//	summarize.go.
+//	summarize.go
+//
+//  Very useful for object triggers
```

Comment:
This should be written in a complete sentence following the style of the other comments.